src/Controls/src/Core/Layout.cs

```diff
@@ -64,7 +64,7 @@ public void Remove(IView child)
 		}
 	}
 
-	public abstract class Layout : View, ILayout, ILayoutController, IPaddingElement
+	public abstract class Layout : View, ILayout, ILayoutController, IPaddingElement, IFrameworkElement
```

Comment:
Should ILayout be an IFrameworkElement?

Reply:
It is but I had to add the interface here so that I could add the explicit implementation of measure lower down in the code